manifests/dragonfly-operator.yaml

```diff
@@ -1978,4 +1978,4 @@ spec:
       securityContext:
         runAsNonRoot: true
       serviceAccountName: dragonfly-operator-controller-manager
-      terminationGracePeriodSeconds: 10
+      terminationGracePeriodSeconds: 10
```

Comment:
same, could you please add the trailing newline?

Reply:
sure, just added them.

Weird, my editor must have removed them?